tasks/__init__.py

```diff
@@ -53,7 +53,7 @@ def decorator(f):
 
 
 @task
-def server(host=None, port=5000, debug=True, live=False, gitlogs=False):
+def server(host="127.0.0.1", port=5000, debug=True, live=False, gitlogs=False):
```

Comment:
It would be fine with me to have these be in a separate PR, but I believe some of them will be needed for this functionality to work.  I started making these changes after talking to Mark while trying to make the server be publicly accessible.  In the end, some other processes could not be made to talk to one another and I ran out of time, so we just made the server run locally and used VNC to view a web page running on the same machine.

After rolling back to that model, I may have left in a change or two that is not strictly required for it to run under those circumstances.  I don't think the adminserver change is needed, and the celery_worker is just a local thing, so those could probably be rolled back.  Perhaps even the main server change isn't needed.  But the one for the apiserver must be made in order for the RMap server to call back to the OSF server to get the data it needs to build the DiSCO.  However, the IP could be made more specific to match the known RMap server location.